response.go

```diff
@@ -178,6 +179,10 @@ func (resp *Response) decodeBody() (err error) {
 				if err = d.Decode(&resp.MetaData); err != nil {
 					return err
 				}
+			case KeyStmtID:
+				if resp.StmtID, err = d.DecodeUint64(); err != nil {
```

Comment:
The response value can be saved in Response.Data.

Reply:
I don't like this approach for some reasons. To fetch this value, we need to get the first element of a slice and cast it to uint64 (or its alias `PreparedStatement`).

Firstly, that makes the business logic of user's application more complicated.
Secondly, I am afraid that user have no idea what for a type the statement id is. If we try to cast it to int or int32 or even int64 the connector will panic. Even if I add the alias to uint64, that will force the user to search this type.

This way I would like to have the returned value as uint64 (or its alias) explicitly anywhere, I'm still considering the alternative approach to return the uint64 instead of Response object.

Reply:
> Firstly, that makes the business logic of user's application more complicated. Secondly, I am afraid that user have no idea what for a type the statement id is. If we try to cast it to int or int32 or even int64 the connector will panic. Even if I add the alias to uint64, that will force the user to search this type.

Maybe this is a reason to hide the details of the implementation and don't return an object of type Response to a client code at all?

Reply:
My point was to return some type explicitly, this way everyone who uses our package will not have to cast this value.
`response.Data` is just a slice of `interface` so we don't have some info about the returning type. With the latest update of the implementation it seems to be minor.